packages/@o3r/core/package.json

```diff
@@ -155,7 +155,7 @@
     "jest": "~29.7.0",
     "jest-junit": "~16.0.0",
     "jest-preset-angular": "~13.1.1",
-    "zone.js": "~0.14.0"
+    "zone.js": "~0.13.1"
```

Comment:
Why should we downgrade zone.js

Reply:
we are still targeting zone.js 0.13, one generatorDep was accidentally updated
I had to remove it to prevent the harmonize to update it everywhere
this should already be fixed by https://github.com/AmadeusITGroup/otter/pull/901